bundles/org.openhab.binding.easee/README.md

```diff
@@ -0,0 +1,170 @@
+
+| Thing/Bridge        | Thing Type          | Description                                                                                   |
+|---------------------|---------------------|-----------------------------------------------------------------------------------------------|
+| bridge              | site                | cloud connection to a site within an Easee account                                            |
+| thing               | charger             | the physical charger which is connected to a circuit within the given site                    |
+| thing               | mastercharger       | like the "normal" charger but with additional capability to control the circuit               |
+
+
+Basically any Easee wallbox that supports the Cloud API should automatically be supported by this binding.
+
+## Discovery
+
+Discovery is implemented and will discover all circuits and chargers assigned to a given site.
+
+## Bridge Configuration
+
+The following configuration parameters are available for the binding/bridge:
+
+| Configuration Parameter | Required | Description                                                                                                                                                                                                                                                                                                                                              |
+|-------------------------|----------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| username                | yes      | The username to login at Easee Cloud service. This should be an e-mail address or phone number.                                                                                                                                                                                                                                                          |
+| passord                 | yes      | Your password to login at Easee Cloud service.                                                                                                                                                                                                                                                                                                           |
+| siteId                  | yes      | The ID of the site containing the wallbox(es) and circuit(s) that should be integrated into openHAB. The ID of your site can be found out via the sites overview (https://easee.cloud/sites). You just need to click on one of the sites listed there, the id will be part of the URL which is then opened. It will be a number with typically 6 digits. |
```

Comment:
```suggestion
| siteId                  | yes      | The ID of the site containing the wallbox(es) and circuit(s) that should be integrated into openHAB. The ID of your site can be found via the sites overview (https://easee.cloud/sites). You just need to click one of the sites listed there, the id will be part of the URL which is then opened. It will be a number with typically 6 digits. |
```

Reply:
fixed